package.json

```diff
@@ -60,7 +60,6 @@
     "flow": "flow",
     "lint": "eslint $(ls src/ | grep '.js$') && echo '# linter passed' && npm run check-license",
     "test": "make test",
-    "test-ci": "npm run test && npm run lint",
```

Comment:
it is used in .travis

Reply:
OK, I'm definitely learning a lot about Travis

Reply:
In the .travis.yml we could change `if [ "$CROSSDOCK" != "1" ]; then npm run test-ci ; fi` to 
`if [ "$CROSSDOCK" != "1" ]; then npm run test ; fi` or for that matter, `if [ "$CROSSDOCK" != "1" ]; then make test ; fi`.

This PR was mostly to check if CI would hang for a PR from a completely new fork. However, `test-ci` script is possibly redundant as it runs `npm run test && npm run lint`; however, `npm run test` runs `make test`, which runs `npm run lint`. Thus, running `npm run lint` again is possibly redundant, hence this whole script is redundant, and one could just run `npm run test`.

Reply:
Good point, there's some clear duplication of work in travis that can be removed. Do you mind repurposing this PR to remove some of the wasted work?

Ideally, for
```
node_js:
- '6'
- '5'
- '4'
- '0.12'
```
We should only run the tests. For
```
- node_js: '0.10'
    env: COVER=1
    sudo: required
    services:
    - docker
```
We should run tests + lint + cover. This should improve travis runtimes. One thing I noticed was that running coveralls actually runs the tests themselves so maybe we can remove the duplication of work there too, but doesn't have to be in this PR since I don't know the work involved.